net-irc/irssi-xmpp/irssi-xmpp-0.54.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="An irssi plugin providing Jabber/XMPP support"
+HOMEPAGE="https://cybione.org/~irssi-xmpp/"
+SRC_URI="https://github.com/cdidier/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	>=net-irc/irssi-1.0.0
+	>=net-libs/loudmouth-1.4.0"
```

Comment:
These versions are always satisfied in ::gentoo tree so the version requirements can be dropped.

Reply:
Modified